tests/micropython/test_cases/test_widgets.py

```diff
@@ -0,0 +1,69 @@
+#! ../../../../../../../lv_micropython/ports/unix/build-standard/micropython ../run_testcase.py
+#LVGL MicroPython binding tester script: Widget-creation/deletion and basic flag/property-settings
+
+
```

Comment:
The purpose of this test would be to see
- if we can create widgets
- deleting widgets doesn't cause crash
- various data types can be passed. For this is recommend setting and getting: 
    - label text (test string)
    - an external array for a chart (integer array)
    - points for a line widget (point array)
    - the map of a button matrix (string array) 
    - options of the dropdown (multi line string) 
    - passing flags and states (enum)

I think having test only for these widgets with a lot of `gc.collect()` calls should be suitable.